sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectAsQuery.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query object either without and clauses or with nonaliased SELECT clause components.
+    /// </summary>
+    public sealed class SelectAsQuery : QueryBase
+    {
+        private FromQuery _fromQuery;
+        private AdtQueryBuilder _parent;
+        private List<string> _clauses;
+
+        internal SelectAsQuery(AdtQueryBuilder parent, FromQuery fromQuery)
+        {
+            _fromQuery = fromQuery;
+            _parent = parent;
+            _clauses = new List<string>();
+        }
+
+        /// <summary>
+        /// Used to alias selectable properties in place of the AS keyword.
```

Comment:
I think this also needs rewording.

Reply:
`Used to rename selectable properties in a query response.`?

Reply:
`Used to select properties with the desired alias`